backend/test_step_time.py

```diff
@@ -1,15 +1,17 @@
 import unittest
 from Step_Time_Calculation import calculate_step_time
 from target_zone_estimation import estimate_target_zone
+import os
 
 THRESHOLD = 20.0 
+DATA_FILE_PATH = os.path.join(os.path.dirname(__file__), "tied belt OSS_f_1.tsv")
 
 class TestStepTimeAndTargetZone(unittest.TestCase):
 
     def test_no_steps_below_threshold(self):
         """Test that all values below threshold yield no step times."""
         force_data = [(0.0, 0), (0.1, 0), (0.2, 10), (0.3, 15)]
-        step_times = calculate_step_time(force_data, THRESHOLD)
```

Comment:
I agree with you on this. `calculate_step_time` already defaults threshold to 20 so there's no point sending threshold if threshold is 20